src/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import classnames from 'classnames';
+import { Todo } from './types/Todo';
+
+type Props = {
+  query: string,
+  setQuery: (query: string) => void,
+  todos: Todo[],
+};
+
+export const Header: React.FC<Props> = ({
+  query,
+  setQuery,
+  todos,
+}) => {
+  const isActicve = todos.filter(todo => !todo.completed);
```

Comment:
```suggestion
  const activeTodos = todos.filter(todo => !todo.completed);
```
use `useMemo` here as well